core-bundle/src/Resources/contao/dca/tl_templates.php

```diff
@@ -504,9 +504,6 @@ public function compareTemplate(DataContainer $dc)
 		$objTemplate->title = StringUtil::specialchars($GLOBALS['TL_LANG']['MSC']['showDifferences']);
 		$objTemplate->charset = System::getContainer()->getParameter('kernel.charset');
 
-		System::getContainer()->setParameter('kernel.debug', false);
-		$GLOBALS['TL_CONFIG']['debugMode'] = false;
```

Comment:
This needs to stay, otherwise `$objTemplate->getResponse()` will return the template content with the debug HTML comments in the `dev` environment.

Reply:
Fixed in 904953456.

Reply:
I can't actually spot any difference with or without `$GLOBALS['TL_CONFIG']['debugMode'] = false;`.

Reply:
> I can't actually spot any difference with or without `$GLOBALS['TL_CONFIG']['debugMode'] = false;`.

With `APP_ENV=dev`?

Reply:
Yes.